integration/apps/sinatra2-classic/Gemfile

```diff
@@ -4,7 +4,7 @@ source "https://rubygems.org"
 
 gem 'puma'
 gem 'unicorn'
-gem 'sinatra'
+gem 'sinatra', '>= 3.0'
```

Comment:
The integration is explicitly named `sinatra2`. Does this version constraint make sense in this context?